yamllint/config.py

```diff
@@ -211,3 +237,20 @@ def get_extended_config_file(name):
 
     # or a custom conf on filesystem?
     return name
+
+
+def _concat_files_content(filenames: list) -> str:
+    """Concatenates file contents together
+
+    Args:
+        filenames (list): the filenames you would like to merge
+
+    Returns:
+        string represents the merged contents of all files
+    """
+    import fileinput
+    content = ""
+    with fileinput.input(filenames) as fin:
+        for line in fin:
+            content += '%s\n' % line
+    return content
```

Comment:
It seems a bit overkill to declare a function just for that, and probably inefficient to concat many `'%s\n' % line`. Plus, all imports must be at top level.

What about something simpler, without creating a new function? E.g.
```python
with fileinput.input(filenames) as f:
    spec = pathspec.PathSpec.from_lines('gitwildmatch', f)
```